README.md

```diff
@@ -53,7 +58,7 @@ Your functions should be organised into a directory with one file acting as the
 import { Hash, encode } from "https://deno.land/x/checksum@1.2.0/mod.ts";
 
 export function make_password_hash(pw: string): string {
```

Comment:
This might be being too anal, but is it worth calling this `make_bad_password_hash`, just to be clear that we're not endorsing doing it this way? Some people on the internet may not identify the light trolling.

Reply:
Yeah sounds good